apps/namehashlabs.org/components/1 - atoms/headline-banner.tsx

```diff
@@ -1,24 +1,25 @@
 import { MegaphoneIcon } from "@heroicons/react/24/outline";
+import { CalendarButton } from "@namehash/internal";
 import { Link } from "@namehash/namekit-react";
 
 export const HeadlineBanner = () => {
   return (
     <div className="px-5 text-white justify-center items-center w-full bg-[#3B0D48] space-x-4 py-4 text-sm relative z-20 flex">
       <div className="flex space-x-3 items-center">
         <MegaphoneIcon className="flex shrink-0 w-6 h-6 opacity-50 -mr-1" />
-        <p className="pr-16 sm:pr-0">
-          Track our progress as an ENS Service Provider
-        </p>
+        <p className=" sm:pr-0">Meet us at DevCon in Bangkok 12-15 Nov 2024</p>
```

Comment:
```suggestion
        <p className=" sm:pr-0">Meet us at DevCon in Bangkok 11-13 Nov 2024</p>
```

We'll be there only 11-13 :+1: